invenio/modules/pages/admin.py

```diff
@@ -60,6 +62,7 @@ class PagesAdmin(ModelView):
 
     form_args = dict(
         template_name=dict(
+            default="default.html",
```

Comment:
I was looking at adding:

```python
default=os.path.basename(cfg["PAGES_DEFAULT_TEMPLATE"])
```

here but then I get:

```
RuntimeError: working outside of application context
```

How can the app config be accessed from the class definition?